nixos/modules/services/web-servers/send.nix

```diff
@@ -159,6 +175,7 @@ in
         LoadCredential = lib.optionalString (
           cfg.redis.passwordFile != null
         ) "redis-password:${cfg.redis.passwordFile}";
```

Comment:
This isn't from this PR, but this snippet should be changed to string concatenation rather than interpolation. If redis.password is a path type (raw path type, not a string), then interpolation will force a store copy, making credentials world-readable.